### DISCUSSION TITLE
Does Hono support for gRPC server/client with Deno runtime?

### DISCUSSION THREAD

**Discussion by xiaojiudev** [honojs/hono#2903] [Q&A]
Hello everyone, I want to implement gRPC server using Deno runtime and Hono framework. Is it possible now?

**Comment by EdamAme-x** [comment]
hmm..
I think `.mount` will be possible, but maybe I don't think that's what you're looking for.

https://github.com/prohazko2/deno-grpc (implement for Deno)

https://zenn.dev/optimisuke/articles/085fdda4770223 (in Japanese)
